pkgs/t3/default.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation rec {
     "VERSION=${version}"
   ];
   nativeBuildInputs = [ help2man ];
-  doCheck = true;
+  doCheck = false;
```

Comment:
> disable `doCheck` for the flake build only, but continuing to run the tests in our own CI builds

It's possible that I misinterpreted this as CI builds of `t3` rather than CI builds of `flox` but I'm not clear how to do the latter and I don't think it makes much practical difference.